src/collectd-btrfs.pod

```diff
@@ -0,0 +1,48 @@
+=encoding UTF-8
+
+=head1 NAME
+
+collectd-btrfs - Documentation of collectd's I<Btrfs plugin>
+
+=head1 SYNOPSIS
+
+LoadPlugin btrfs
+<Plugin btrfs>
+  Path "/tmp/btrfstest"
+</Plugin>
+
+
+=head1 DESCRIPTION
+
+The btrfs plugin read the device stats from an mounted BTRFS-Filesystem.
+
+=head1 CONFIGURATION
+
+Here is a configuration example to get you started. Read below for more
+information.
+
+LoadPlugin btrfs
+<Plugin btrfs>
+  Path "/tmp/btrfstest"
+</Plugin>
+
+
+LoadPlugin btrfs
```

Comment:
Why two configuration examples are here?

Reply:
Well, just to demonstrate that the Path-Option can be used multiple times.